src/pastel/PastelSig.re

```diff
@@ -13,6 +13,7 @@ module type PastelSig = {
   let setMode: mode => unit;
   let useMode: (mode, unit => 'a) => 'a;
   let length: string => int;
+  /* probably don't want to release this*/
```

Comment:
Maybe update the comment stating this is intentionally not released until we finalize the API